<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraInfoWidget</class>
 <widget class="QWidget" name="CameraInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraInfo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>241</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="camera_name">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="camera_chipsize_label">
     <property name="text">
      <string>Chip Size (mm)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>191</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="camera_chip_size">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="camera_pixelsize_label">
     <property name="text">
      <string>Pixels Size (um)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>191</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="camera_pixels_size">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="camera_resolution_label">
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>191</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="camera_resolution">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="camera_bpp_label">
     <property name="text">
      <string>bpp</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>241</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="camera_bpp">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QWidget" name="chipInfo" native="true"/>
   </item>
   <item row="6" column="1" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>47</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>chipInfo</zorder>
  <zorder>camera_name</zorder>
  <zorder>horizontalSpacer_5</zorder>
  <zorder>camera_chipsize_label</zorder>
  <zorder>label</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>horizontalSpacer_4</zorder>
  <zorder>horizontalSpacer_3</zorder>
  <zorder>camera_bpp</zorder>
  <zorder>camera_resolution</zorder>
  <zorder>camera_bpp_label</zorder>
  <zorder>camera_chip_size</zorder>
  <zorder>camera_pixels_size</zorder>
  <zorder>camera_pixelsize_label</zorder>
  <zorder>camera_resolution_label</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
